charts/kube-master/templates/api.yaml

```diff
@@ -165,6 +165,15 @@ spec:
             - mountPath: /etc/kubernetes/bootstrap
               name: bootstrap
               readOnly: true
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 6443
+              scheme: HTTPS
```

Comment:
Does this work for the self generated cert here? Does the health check run with `insecure` by default?

Reply:
seems like it does

Reply:
Indeed, this is interesting. It also works for readiness probe, see https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes

> If scheme field is set to HTTPS, the kubelet sends an HTTPS request skipping the certificate verification.